tools/StaticAnalysis/ExampleAnalyzer/Measure-MarkdownOrScript.ps1

```diff
@@ -9,20 +27,21 @@
 
 [CmdletBinding(DefaultParameterSetName = "Markdown")]
 param (
-    [Parameter(Mandatory, HelpMessage = "Markdown searching paths. Empty for current path. Supports wildcard.", ParameterSetName = "Markdown")]
+    [Parameter(Mandatory, ParameterSetName = "Markdown")]
     [AllowEmptyString()]
     [string[]]$MarkdownPaths,
-    [Parameter(Mandatory, HelpMessage = "PowerShell scripts searching paths. Empty for current path. Supports wildcard.", ParameterSetName = "Script")]
+    [Parameter(Mandatory, ParameterSetName = "Script")]
     [AllowEmptyString()]
-    [string[]]$ScriptPaths,
-    [Parameter(HelpMessage = "PSScriptAnalyzer custom rules paths. Empty for current path. Supports wildcard.")]
+    [string[]]$ScriptPath,
     [string[]]$RulePaths,
+    [Parameter(Mandatory, ParameterSetName = "Script")]
     [string]$CodeMapPath,
```

Comment:
Code map should be transparent to users so it should be optional.
As a user, I only want to know the error is on which line. I don't care where the code map file is.